ceremonies/src/lib.rs

```diff
@@ -292,20 +292,20 @@ decl_module! {
             let sender = ensure_signed(origin)?;
 
             ensure!(<encointer_communities::Module<T>>::community_identifiers().contains(&cid),
-                "CommunityIdentifier not found");
+                Error::<T>::InexistentCommunity);
 
             ensure!(<encointer_communities::Module<T>>::bootstrappers(&cid).contains(&sender),
-            "only bootstrappers can endorse newbies");
+            Error::<T>::AuthorizationRequired);
 
             ensure!(<BurnedBootstrapperNewbieTickets<T>>::get(&cid, &sender) < AMOUNT_NEWBIE_TICKETS,
-            "bootstrapper has run out of newbie tickets");
+            Error::<T>::OutOfNewbieTickets);
```

Comment:
generally I find that the naming lacks a convention.. some are named as exceptions, some read like prose.. some functions have an error for every possible ensure! macro, some use a broader context (e.g. AttendanceUnverifiedOrAlreadyUsed)

Reply:
I actually don't quite agree. I think they are all rather formulated as exceptions. 

I agree about the context discrepancies, but in case of `AttendanceUnverifiedOrAlreadyUsed`, we don't have more granular information to give.